src/controllers/commentsController.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-irregular-whitespace */
+import models from '../database/models';
+import responseUtil from '../utils/responseUtil';
+import strings from '../utils/stringsUtil';
+
+export default class CommentsController {
+
+  static async addComment(req, res) {
+    const { comment } = req.body;
+    const { id } = req.user.payload;
+    const requestId = parseInt(req.params.id, 10);
+    try {
+      const newComment = { comment, userId: id, requestId };
+      const addedComment = await models.comments.create(newComment);
+      const {
+        deleted, createdAt, updatedAt, userId,
+        ...data
+      } = addedComment.dataValues;
+
+      return responseUtil(res, 200, strings.request.success.SUCCESS_ADD_COMMENT, data);
+    } catch (error) {
+      return res.send(error);
+    }
+  }
+
+
+  static async getComments(req, res) {
+    try {
+      const requestId = parseInt(req.params.id, 10);
+      const comments = await models.comments.findAll({
+        where: { requestId }
```

Comment:
This gets all comments on a request even the soft deleted ones. Add a condition to get comments only when the `deleted= false`